packages/core/src/form-field/FormLabel.css

```diff
@@ -70,3 +70,7 @@
   margin-left: 6px;
   vertical-align: top;
 }
+
+.backwardsCompat .uitkFormLabel {
```

Comment:
has a space here because it is used in more than 1 component

Reply:
Preferably list them out. 
If there is a lot, then put the comment in code.